exasol_integration_test_docker_environment/cli/commands/health.py

```diff
@@ -16,36 +12,11 @@ def health():
     If no issues have been found, using the library or executing the test should work just fine.
     For all found issues there will be a proposed fix/solution.
 
-    If the environment was found to be healthy the exit code will be 0.
+    If the environment was found to be healthy the exit code will be 0, otherwise -1.
     """
     success, failure = 0, -1
-
-    problems = set(health_checkup())
-    if not problems:
+    message = api.health()
```

Comment:
Actually, I started to do that. But then I thought that it would be for a client quite elaborate to handle all possible error cases.  Normally, the clients are not interested in the details but only would show the error string.

Reply:
also somewhat true, maybe it is actually better to throw an exception in the API with the message and in the CLI catch it and print the message

Reply:
Done